dashboard/config/scripts/allthethings.script

```diff
@@ -156,10 +157,10 @@ level 'Test longassessment levelgroup split new', assessment: true
 stage 'Star wars'
 skin 'hoc2015'
 video_key_for_next_level 'starwars_events'
-level_concept_difficulty '{}'
+level_concept_difficulty '{"sequencing":1}'
```

Comment:
What does this change do?

Reply:
Oh weird. Must be some sort of merge issue. I'll look into what happened here.

Reply:
Removed